drivers/hmis/app/models/hmis/user.rb

```diff
@@ -85,12 +90,12 @@ def lock_access!(opts = {})
   end
 
   private def permissions_base_for_entity(entity)
-    return [entity.data_source, *entity.projects] if entity.is_a? Hmis::Hud::Client
+    return entity.projects if entity.is_a? Hmis::Hud::Client
     return entity if entity.is_a? Hmis::Hud::Organization
     return entity if entity.is_a? Hmis::Hud::Project
 
     if entity.is_a? Hmis::File
-      return [entity.client.data_source, *entity.client.projects] unless entity.enrollment.present?
+      return entity.client.data_source unless entity.enrollment.present?
```

Comment:
I don't understand this change, can you explain?

If a user has access to see a client BECAUSE they can see a project that the client is enrolled in, then they should be able to see that client's files. I think this change takes that capability away. Is that right?

Reply:
Yeah, I think this needs another look. I forgot that files are not connected to clients via enrollments. The intention was to base the permissions in the same way as client, so I'll go back and adjust to make that the case

Reply:
Ok, then I think this should be `entity.client.projects_including_wip`